torchvision/models/_api.py

```diff
@@ -140,3 +160,59 @@ def _get_enum_from_fn(fn: Callable) -> WeightsEnum:
         )
 
     return cast(WeightsEnum, weights_enum)
+
+
+M = TypeVar("M", bound=Type[nn.Module])
+
+BUILTIN_MODELS = {}
```

Comment:
Nit: someone could use this mechanism to register their own models. Should we rename this to available_models or something similar?

Reply:
Happy to make the change to `AVAILABLE_MODELS`. The reason why I used this naming is to align with the ones we use on Datasets:
https://github.com/pytorch/vision/blob/6ca9c76adb6daf2695d603ad623a9cf1c4f4806f/torchvision/prototype/datasets/_api.py#L12

https://github.com/pytorch/vision/blob/6ca9c76adb6daf2695d603ad623a9cf1c4f4806f/torchvision/prototype/datasets/_api.py#L23

Do you think we should make the change now or postpone for when/if we decide to open-up the registration mechanism?

Reply:
In that case we can keep it consistent and make the changes later in both should we decide to